src/world.h

```diff
@@ -0,0 +1,26 @@
+/****************************************************************************/
+//    Copyright (C) 2009 Aali132                                            //
+//    Copyright (C) 2018 quantumpencil                                      //
+//    Copyright (C) 2018 Maxime Bacoux                                      //
+//    Copyright (C) 2020 Julian Xhokaxhiu                                   //
+//    Copyright (C) 2020 myst6re                                            //
+//    Copyright (C) 2020 Chris Rizzitello                                   //
+//    Copyright (C) 2020 John Pritchard                                     //
+//    Copyright (C) 2020 Marcin Gomulak                                     //
```

Comment:
As you'll be part of this project starting this PR, please ensure your name is present everywhere this header is. I'll work on streamlining the injection of it in the source files somehow via CMake. Until then, you'd have to handle this manually. Painful process, I know...